### ISSUE TITLE
Fire on coins is not circular

### ISSUE THREAD

**Issue by ncrecc** [TerryCavanagh/diceydungeons.com#2533]
![image](https://user-images.githubusercontent.com/11216597/134751062-8e4b9cff-9e9c-4501-b5fe-e35151e61d60.png)

**Comment by TerryCavanagh** [comment]
Yeah, this one's probably not happening - huge hassle to do for complicated reasons!
